src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs

```diff
@@ -78,26 +78,20 @@ private static void TestSimpleParameter_Type(object paramValue, string expectedT
             string procName = DataTestUtility.GetUniqueNameForSqlServer("paramProc1");
             try
             {
-                using (SqlConnection conn = new SqlConnection(s_connStr))
-                {
-                    conn.Open();
-                    DropStoredProcedure(conn, procName);
-                    xsql(conn, string.Format("create proc {0} (@param {1}) as begin select @param end;", procName, expectedBaseTypeName));
-
-                    using (SqlCommand cmd = conn.CreateCommand())
-                    {
-                        cmd.CommandText = procName;
-                        cmd.CommandType = CommandType.StoredProcedure;
-                        SqlParameter p = cmd.Parameters.AddWithValue("@param", paramValue);
-                        cmd.Parameters[0].SqlDbType = GetSqlDbType(expectedBaseTypeName);
-                        using (SqlDataReader dr = cmd.ExecuteReader())
-                        {
-                            dr.Read();
-                            VerifyReaderTypeAndValue("Test Simple Parameter [Data Type]", expectedBaseTypeName, expectedTypeName, dr[0], expectedTypeName, paramValue);
-                            dr.Dispose();
-                        }
-                    }
-                }
+                using SqlConnection conn = new(s_connStr);
+                conn.Open();
+                DropStoredProcedure(conn, procName);
+                xsql(conn, string.Format("create proc {0} (@param {1}) as begin select @param end;", procName, expectedBaseTypeName));
+
+                using SqlCommand cmd = conn.CreateCommand();
+                cmd.CommandText = procName;
+                cmd.CommandType = CommandType.StoredProcedure;
+                SqlParameter p = cmd.Parameters.AddWithValue("@param", paramValue);
+                cmd.Parameters[0].SqlDbType = GetSqlDbType(expectedBaseTypeName);
+                using SqlDataReader dr = cmd.ExecuteReader();
+                dr.Read();
+                VerifyReaderTypeAndValue("Test Simple Parameter [Data Type]", expectedBaseTypeName, expectedTypeName, dr[0], expectedTypeName, paramValue);
+                dr.Dispose();
```

Comment:
I definitely agree. It was in the code to dispose it and I was planning to change that in the next PR, but since you mentioned that I will do it in this PR.